src/components/NewDesign/Swap/transactionDetails/TransactionDetails.tsx

```diff
@@ -0,0 +1,32 @@
+import { printBN } from '@consts/utils'
+import { Grid, Typography } from '@material-ui/core'
+import BN from 'bn.js'
+import React from 'react'
+import { useStyles } from './style'
+
+interface IProps {
+  open: boolean
+  pool: {val: BN, scale: number}
+}
+
+const TransactionDetails: React.FC<IProps> = ({ open, pool }) => {
+  const classes = useStyles()
+  return (
+    <Grid container className={classes.transactionDetailsInfo} style={{ opacity: open ? '1' : '0', zIndex: open ? 1 : 0 }}>
+      <Grid className={classes.detailsInfoWrapper}>
+        <Typography component='h2'>Transaction details</Typography>
+        <Typography component='p'>
+                Fee: <Typography component='span'>
+            {printBN(pool.val, pool.scale)} %
+          </Typography>
+        </Typography>
+        <Typography component='p'>
+                Exchange rate: <Typography component='span'>
+            {printBN(pool.val, pool.scale)} xUSD
```

Comment:
don't use the same value for fee and exchange rate

Reply:
done